teuthology/orchestra/remote.py

```diff
@@ -82,6 +82,17 @@ def hostname(self):
         return self._hostname
 
     @property
+    def machine_type(self):
+        if not getattr(self, '_machine_type', None):
+            remote_info = ls.get_status(self.hostname)
+            if not remote_info:
+                log.info(
+                    "Failed to query lock server for status of {remote}".format(remote=self))
```

Comment:
Agreed.  I just made that change and moved this logging statement to teuthology.lockstatus